From 5eebf47c599b6631a0046554d870e370d1541858 Mon Sep 17 00:00:00 2001
From: Jiang Lu <lu.jiang@windriver.com>
Date: Wed, 14 Nov 2018 15:49:46 +0800
Subject: [PATCH] soc:fsl:Add missing reference for Kconfig.arm

Add missing reference for Kconfig.arm in soc/fsl/Kconfig.

Signed-off-by: Jiang Lu <lu.jiang@windriver.com>
---
 drivers/soc/fsl/Kconfig |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/soc/fsl/Kconfig b/drivers/soc/fsl/Kconfig
index 7a9fb9b..6a6f75b 100644
--- a/drivers/soc/fsl/Kconfig
+++ b/drivers/soc/fsl/Kconfig
@@ -16,3 +16,7 @@ config FSL_GUTS
 	  Initially only reading SVR and registering soc device are supported.
 	  Other guts accesses, such as reading RCW, should eventually be moved
 	  into this driver as well.
+
+if ARM || ARM64
+source "drivers/soc/fsl/Kconfig.arm"
+endif
-- 
1.7.9.5

